src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -252,8 +252,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
     with self.argument_context('storage account blob-inventory-policy create') as c:
         c.argument('policy', type=file_type, completer=FilesCompleter(),
                    help='The Storage Account Blob Inventory Policy, string in JSON format or json file path. See more '
-                   'details in https://review.docs.microsoft.com/en-us/azure/storage/blobs/blob-inventory#'
-                   'inventory-policy.')
+                   'details in https://docs.microsoft.com/en-us/azure/storage/blobs/blob-inventory#inventory-policy.')
```

Comment:
```suggestion
                   'details in https://docs.microsoft.com/azure/storage/blobs/blob-inventory#inventory-policy.')
```
Don't specify language and let docs decide accordingly